upgrade-notes/backend_20241022_082619.md

```diff
@@ -0,0 +1,27 @@
+#### use moved Ready Category Seo Mix code from packages ([#3494](https://github.com/shopsys/shopsys/pull/3494))
+
+-   [features moved](#movement-of-features-from-project-base-to-packages) from project-base to the frontend-api and framework packages:
```

Comment:
as I mentioned somewhere at the beginning of the review, we should IMHO also mention the new `DeleteReadyCategorySeoMixFacade` with extracted `deleteAllWithParameter()` method

Reply:
added in 0ea2e69645